Fix grammar in DG (#524)
AY2223S1-CS2103T-W16-2/tp: docs/_dg/NFRs.md

<!-- markdownlint-disable-file first-line-h1 -->
1. The product should work on any mainstream OS (Windows, macOS, Linux) that has Java 11 or above installed.
1. The product should be able to hold up to 1000 items without noticeable sluggishness in performance for typical usage.

> 1. A user with above average typing speed for regular English text (i.e. not code or system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.

Reviewer: Phrasing could be better I think.

```suggestion
1. A user with above average typing speed for regular, English text (not code, nor system admin commands) should be able to accomplish most of the tasks faster using commands than using the mouse.
```

Author: Okay
